cpp/src/arrow/vendored/datetime/tz.cpp

```diff
@@ -3597,19 +3597,17 @@ tzdb::locate_zone(const std::string& tz_name) const
 {
```

Comment:
This is a vendored file (as the file path suggests), so we shouldn't do such changes to it. Can you revert them?

Reply:
done.